homeassistant/components/switch/homekit_controller.py

```diff
@@ -62,3 +70,20 @@ def turn_off(self, **kwargs):
                             'iid': self._chars['on'],
                             'value': False}]
         self.put_characteristics(characteristics)
+
+    @property
+    def state_attributes(self):
```

Comment:
This should be `device_state_attributes`.

Reply:
Ok so base entities are using state_attributes and platforms like homekit should use device_state_attributes. Is that correct ?

Reply:
Thank you very much for clarification. I am just learning how to contribute and use things properly. I appreciate your time and help :)